charts/victoria-metrics-operator/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Next release
 
 - Added ability to override deployment namespace using `namespaceOverride` and `global.namespaceOverride` variables
+- Fixed template for cert-manager certificates
+- Removed cluster role disabling, when operator watches self namespace
```

Comment:
But we don't need clusterrole&clusterrolebinding when operator watches only one ns.

Reply:
setting permissions for `nonResourceURLs` requires having clusterrole

Reply:
`nonResourceURLs` was mistakenly put into the Role resource, please check https://github.com/VictoriaMetrics/helm-charts/pull/1457/commits/6d270532260286c59167b58c7abe904908c85b4b